examples/interpretation/notebooks/utilities.py

```diff
@@ -424,6 +425,11 @@ def download_pretrained_model(config):
     # Update config MODEL.PRETRAINED
     # TODO: Only HRNet uses a pretrained model currently.
     # issue https://github.com/microsoft/seismic-deeplearning/issues/267
+
+    # now that we have a pre-trained model, we can set it
+    if "PRETRAINED" not in config.MODEL.keys():
+        config.MODEL["PRETRAINED"] = "dummy"
+
     opts = [
```

Comment:
Is it true that only HRnet has a pretrained model? 
Also, is "dummy" here like a placeholder?

Reply:
it's a key placeholder because we need to add a PRETRAINED key - we check the path later in the code so this can literally be anything